changelog/fragments/rescue-reconciliation.yaml

```diff
@@ -0,0 +1,8 @@
+entries:
+  - description: >
+      Stop reconciling tasks when the event raised is a rescue in Ansible-based Operators. 
+      More info: [Bugzilla 1856714](https://bugzilla.redhat.com/show_bug.cgi?id=1856714)
+
+    kind: "bugfix"
+
+    breaking: false
```

Comment:
Since it was not added in the same PR we need to fill : pull_request_override:  with the PR number.